### ISSUE TITLE
Can this be ported to React Native?

### ISSUE THREAD

**Issue by mafiusu** [weknowinc/react-bubble-chart-d3#27]
Hello I just wondered if this library can be ported to React Native and if yes? What should I consider if I do?

**Comment by jmsv23** [comment]
This library use D3 for the graphics, on react native the svg's are handle in a particular way and the styles are different that the web standar,  i am not entirely sure if D3 play nice with react native, but you can investigate the way to display graphics inside of a webview and use this library

**Comment by PotLid** [comment]
You can refer the structural design of React JS, but need to change its component into targeted devices compatible component in React Native. 
[https://facebook.github.io/react-native/docs/components-and-apis.html](url)

Check the React Native's API DOC

**Comment by mafiusu** [comment]
@jmsv23 @PotLid  I'll definitely look into it since [React Native supports](https://github.com/react-native-community/react-native-svg#supported-elements) almost all kind of svg elements ( Svg, Rect, Circle, Ellipse, Line, Polygon, Polyline, Path, Text, TSpan, TextPath, G, Use, Symbol, Defs, Image, ClipPath, LinearGradient, RadialGradient, Mask & Pattern)

**Comment by PotLid** [comment]
@mafiusu Yes, Probably compatible elements can be used under Native Env. without any kinds of heavy modifications, yet If you use JSX (HTML) syntax which is not compatible, you need to fix it later on accordingly to the Native Env. 

p.s If your issue is resolved, please close the issue. If it isn't leave comments in the issue section, thanks.
